pandas/core/generic.py

```diff
@@ -7346,6 +7346,12 @@ def clip(
             If False: show all values for categorical groupers.
 
             .. versionadded:: 0.23.0
+        dropna : bool, default True
+            If True, and if group keys contain NaN values, NaN values together
```

Comment:
don't say NaN, say NA values (e.g. can also be NaT or the new NA scalar)

Reply:
changed!